.github/cspell.yaml

```diff
@@ -0,0 +1,11 @@
+version: '0.2'
+import:
+  - ../cspell.yaml
+words:
+  - commitish
+  - endgroup
+  - lastexitcode
+  - longpaths
```

Comment:
Arent these compound words that should be handled automatically now?

Reply:
Good catch. I'll verify and remove those. I made this file before we turned on allowing compound words yesterday.